htdocs/commande/list.php

```diff
@@ -784,11 +785,14 @@
 		$text_warning='';
 		$nbprod=0;
 
-		$companystatic->id=$obj->socid;
+        $companystatic->id = $obj->socid;
 		$companystatic->code_client = $obj->code_client;
-		$companystatic->name=$obj->name;
-		$companystatic->client=$obj->client;
-		$companystatic->email=$obj->email;
+		$companystatic->name = $obj->name;
+		$companystatic->client = $obj->client;
+		$companystatic->email = $obj->email;
+		if (!isset($cacheGetNomUrl[$obj->socid])) {
```

Comment:
I think this must be 
if (!isset($getNomUrl_cache[$obj->socid])) {

Reply:
oups...
done
tks for your help